safe_transaction_service/account_abstraction/tests/test_models.py

```diff
@@ -0,0 +1,5 @@
+from django.test import TestCase
+
+
+class TestModels(TestCase):
+    pass
```

Comment:
We are not including too much tests, I would agree if you create an issue to do it later in other PR.

Reply:
Done, included more tests